src/silx/gui/plot3d/ParamTreeView.py

```diff
@@ -222,14 +216,39 @@ def __init__(self, parent=None):
 class BooleanEditor(qt.QCheckBox):
     """Checkbox editor for bool.
 
-    This is a QCheckBox with white background.
+    Wrap a QCheckBox to define a different user property with `clicked` signal.
```

Comment:
This was needed since by default is reacts on the `toggled` signal: This causes issues with Qt6 because the value is set in the editor **before** the editor is added to the the tree view... 🤷